File: Iniciantes/1145.py
entrada = raw_input().split(" ")
msg = ""
i = 1
cont = 0
while i <=  int(entrada[1]):
	msg = msg + " " + str(i)
	cont = cont + 1
	if cont == 3:
		print msg
		cont = 0
		msg = ""
	i = i + 1

File: Iniciantes/1145.c
#include <stdio.h>

int main(void) {
	int x, y, i = 1, cont = 0;

	scanf("%d %d", &x, &y);

	while (i <= y) {
		printf("%d", i);
		cont++;
		if (cont == x) {
			printf("\n");
			cont = 0;
		} else
			printf(" ");
		i++;
	}
return 0;
}